vitrina/datasets/templates/vitrina/datasets/datatable.html

```diff
@@ -0,0 +1,88 @@
+{% load i18n %}
+
+<table class="table is-striped is-hoverable is-fullwidth no-margin-bottom table-smaller-padding" id="resource-table">
+    <thead>
+    <tr>
+        <th></th>
+        <th class="has-text-centered">{% translate "Versija" %}</th>
+        <th>{% translate "Teritorija" %}</th>
+        <th>{% translate "Pavadinimas" %}</th>
+        <th>{% translate "Periodo pradžia" %}</th>
+        <th>{% translate "Periodo pabaiga" %}</th>
+        <th>{% translate "Dydis" %}</th>
+        <th class="has-text-centered">{% translate "Formatas" %}</th>
+        <th>{% translate "Publikuota" %}</th>
+        <th>{% translate "Atnaujinta" %}</th>
+        <th></th>
+    </tr>
+    </thead>
+    <tbody>
+    {% for r in resources %}
+        <tr>
+            <td><span class="icon icon-is-small">
+                {% if r.type == "URL" %}
```

Comment:
```suggestion
                {% if r.is_external_url %}
```

Do not hardcode constants, use a model method instead.